src/test/java/umk/mat/pajda/ProjektZespolowy/Services/AuthenticationServiceImplTest.java

```diff
@@ -0,0 +1,136 @@
+package umk.mat.pajda.ProjektZespolowy.Services;
+
+import static org.mockito.ArgumentMatchers.any;
+
+import java.util.Optional;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import umk.mat.pajda.ProjektZespolowy.DTO.RefreshTokenDTO;
+import umk.mat.pajda.ProjektZespolowy.DTO.UserDTO;
+import umk.mat.pajda.ProjektZespolowy.entity.User;
+import umk.mat.pajda.ProjektZespolowy.misc.UserConverter;
+import umk.mat.pajda.ProjektZespolowy.repository.UserRepository;
+import umk.mat.pajda.ProjektZespolowy.services.JWTService;
+import umk.mat.pajda.ProjektZespolowy.services.impl.AuthenticationServiceImpl;
+
+public class AuthenticationServiceImplTest {
+
+  @Mock private UserRepository userRepository;
+  @Mock private UserConverter userConverter;
+
+  @Mock private PasswordEncoder passwordEncoder;
+
+  @Mock private AuthenticationManager authenticationManager;
+
+  @Mock private JWTService jwtService;
+
+  @InjectMocks private AuthenticationServiceImpl authenticationService;
+
+  @BeforeEach
+  public void setUp() {
+    MockitoAnnotations.openMocks(this);
+  }
+
+  @Test
+  public void shouldSuccesWhenGetUserTest() {
+    String mail = "test@test.com";
+    User user = new User();
+    UserDTO userDTO = new UserDTO();
+    user.setMail(mail);
+    userDTO.setMail(mail);
+    Mockito.when(userRepository.findByMail(mail)).thenReturn(Optional.of(user));
+    Mockito.when(userConverter.createDTO(user)).thenReturn(userDTO);
+    Assertions.assertEquals(authenticationService.getUser(mail), userDTO);
+  }
+
+  @Test
+  public void shouldNullWhenUserNotFoundTest() {
```

Comment:
what does null mean here? We could use more observability focused test description here

Reply:
something like i added ?

Reply:
something like what i added ?